.rubocop.yml

```diff
@@ -329,3 +329,18 @@ RSpec/SubjectStub:
 # the default style recently changed, but it's not worth changing
 Rails/FilePath:
   EnforcedStyle: arguments
+
+Lint/RaiseException:
+  Enabled: false
+
+Lint/StructNewOverride:
+  Enabled: false
+
+Style/HashEachMethods:
+  Enabled: false
+
+Style/HashTransformKeys:
+  Enabled: false
+
+Style/HashTransformValues:
+  Enabled: false
```

Comment:
Thoughts on enabling these cops but adding the exceptions to the `.rubocop_todo.yml` so that they have a better chance of being looked at some day?

Reply:
@omgitsbillryan honestly i didn't read enough about them to know what they're for yet. Intro'd in v0.80 and v0.81 in the new way rubocop is handling additions. Probably worth actively deciding our thoughts on each one so we don't have todos we don't want to ever do :)

Reply:
Mostly this was to get rid of the extra output